tests/php/Shortcodes/ImageShortcodeProviderTest.php

```diff
@@ -46,33 +47,31 @@ protected function tearDown(): void
     public function testShortcodeHandlerDoesNotFallBackToFileProperties()
     {
         $image = $this->objFromFixture(Image::class, 'imageWithTitle');
-        $parser = new ShortcodeParser();
-        $parser->register('image', [ImageShortcodeProvider::class, 'handle_shortcode']);
-
-        $this->assertEquals(
-            sprintf(
-                '<img src="%s" alt="">',
-                $image->Link()
-            ),
-            $parser->parse(sprintf('[image id=%d]', $image->ID))
```

Comment:
Most of these test were throwing errors because `ImageManipulation` was returning attributes in a different order than.

`ImageManipulation` does a few things a bit differently which also led to slightly different result in some context.

Because none of these test actually included messages, it was really hard to figure out what each test was meant to be doing. 

So I basically ended rewriting all the test with more descriptive message and to uso a utility assert method.